### PR TITLE
Upgrade server to the new version easily

### PR DESCRIPTION
**Pull request by muratugureminoglu**
https://github.com/ant-media/Ant-Media-Server/issues/5833

### CODE REVIEW

File: src/main/server/update.sh [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+# Description:
+#   This script facilitates the seamless update of Ant Media Server to the latest version,
+#   ensuring you have the most up-to-date features, improvements, and bug fixes. 
+#   For more information: https://github.com/ant-media/Ant-Media-Server/releases
+
+# Usage:
+#   1. Run the script directly in the terminal using "./update.sh".
+#   2. Utilize the dashboard's integrated update feature for a user-friendly experience.
+
 INSTALL_DIRECTORY=/usr/local/antmedia
```
Comment (mekya) [inline src/main/server/update.sh:21]: I think install directory may be different, just get the directory where update.sh is.

File: src/main/server/update.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#   2. Utilize the dashboard's integrated update feature for a user-friendly experience.
+
+# Note:
+#   - Ensure internet connectivity for downloading the latest Ant Media Server release.
+
+# -----------------------------------------------------------------------------
+
+INSTALL_DIRECTORY=/usr/local/antmedia
+
+REMOTE_VERSION=$(curl -s "https://antmedia.io/rest/VERSION")
```
Comment (mekya) [inline src/main/server/update.sh:23]: This looks fragile to me. I think we need to have more robust new version checking structure while we upload the latest version to antmedia.io

File: src/main/server/update.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+LOCAL_VERSION=$(unzip -p $INSTALL_DIRECTORY/ant-media-server/ant-media-server.jar | grep -a "Implementation-Version"|cut -d' ' -f2 | tr -d '\r')
```
Comment (mekya) [inline src/main/server/update.sh:24]: I think the path in ant-media-server/ant-media-server.jar not correct.

**Review by mekya — CHANGES_REQUESTED** [review]
Thank you for the development @muratugureminoglu 

I've just added some comments. 

I am eager to see the CI tests in Scripts repository. Let me know when it's ready to see.

File: src/main/server/update.sh (hunk 3 above)
Comment (muratugureminoglu) [reply src/main/server/update.sh:24]: Yes, It 's my bad. Sorry for that. I m still making some changes.

File: src/main/server/update.sh (hunk 2 above)
Comment (muratugureminoglu) [reply src/main/server/update.sh:23]: CI creates the VERSION file. If it is not suitable, I may consider different things.

File: src/main/server/update.sh (hunk 1 above)
Comment (muratugureminoglu) [reply src/main/server/update.sh]: Fixed it.

**Comment by mekya** [comment]
Hi @muratugureminoglu ,

I think you just need to update this PR according to the info below. 

Btw, latest-snapshot is available here -> https://antmedia.io/download/latest-snapshot.json
latest-version is going to be available here when it's released in the same format with latest-snapshot.json-> https://antmedia.io/download/latest-version.json

You can use these file to compare the versions.

Secondly, CI test is important. I need to see the CI test before merging this PR.

**Comment by muratugureminoglu** [comment]
Hi @mekya 

The first one is done. 

Second one, https://github.com/ant-media/Scripts/pull/273/files

**Review by mekya — APPROVED** [review]

**Pull request merged**